index.js

```diff
@@ -46,6 +48,22 @@ module.exports = class TrailsApp extends events.EventEmitter {
     this.setMaxListeners(app.config.main.maxListeners || 128)
   }
 
+  buildLog(config) {
+    const logger = config.log.logger
+    if (logger.levels) {
```

Comment:
yeah my point was if user use winston, we add the possibility to do `app.log()` and if they are using another logger we just let as it. But i will be better if it can work for all logger you're right. What I'm thinking right now is we can add `config.log.defaultLevel`, take winston levels as defaults levels and map them to Trails log object if those exists. 
And maybe give the possibility to override those levels with `config.log.levels`.
I can change my code to reflect this if you prefer this way.